cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -854,3 +854,84 @@ def _revert():
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.credentials.get_text_of("Vault") == vault_creds.name
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.meta(automates=[BZ(1444092)])
+@pytest.mark.parametrize(
+    "credential", CREDENTIALS, ids=[cred[0] for cred in CREDENTIALS]
+)
+@pytest.mark.provider(
+    [RHEVMProvider, EC2Provider, VMwareProvider, AzureProvider], selector=ONE_PER_TYPE
+)
+@pytest.mark.uncollectif(
+    lambda credential, provider: not (
+        (credential[0] == "Amazon" and provider.one_of(EC2Provider))
+        or (credential[0] == "VMware" and provider.one_of(VMwareProvider))
+        or (
+            credential[0] == "Red Hat Virtualization" and provider.one_of(RHEVMProvider)
+        )
+        or (credential[0] == "Azure" and provider.one_of(AzureProvider))
+    )
+)
+@pytest.mark.tier(3)
+def test_ansible_service_cloud_credentials(
+    appliance,
+    request,
+    ansible_catalog_item,
+    ansible_service,
+    ansible_service_catalog,
+    credential,
+    provider_credentials,
+):
+    """
+        When the service is viewed in my services it should also show that the cloud credentials
+    were attached to the service.
+
+    Bugzilla:
+        1444092
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/4h
+        tags: ansible_embed
+    """
+    playbook = credential[2]
+    with update(ansible_catalog_item):
+        ansible_catalog_item.provisioning = {
+            "playbook": playbook,
+            "cloud_type": provider_credentials.credential_type,
+            "cloud_credential": provider_credentials.name,
+        }
+
+    @request.addfinalizer
+    def _revert():
+        with update(ansible_catalog_item):
```

Comment:
Why is this required ? The ansible_catalog_item fixture is deleting the catalog item after yield .
https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/ansible_fixtures.py#L81

Reply:
`ansible_catalog_item` fixture will be used in different tests. So, resetting back to default values to use it as default fixture for next tests.

Reply:
I don't think this is a good practice, as if this fails it will potentially impact all following test cases, and be difficult to debug.  If you are modifying the catalog item, it should be made function scoped, at least for this test case.

Reply:
@mshriver This will impact on Over all tests execution in terms of time. Also, in most of the tests `ansible_catalog_tem` is being modified to use another playbook. Does that makes sense.?

Reply:
@mshriver This PR --> https://github.com/ManageIQ/integration_tests/pull/9490 will take care of this.

Reply:
Also, will push changes to #9490